### PR TITLE
Backwards compatibility

### PR DESCRIPTION
**Pull request by anthonycr**
#### Summary
Due to a problem where we didn't use the stag serializer in some code, the field names were all broken because of the `@GsonAdapterKey` annotation. Since we have now replaced that annotation with the `@SerializedName` annotation which is picked up by gson regardless of whether the stag plugin is used, we need to add backwards compatibility for the broken cached json. The only classes that were affected were those that are referenced by `Video` and `VideoFile`.

#### Implementation Summary
By using the alternate name scheme, we can add these optional names as json keys to pick up what was broken. When these classes are serialized again, the `value` is used and the `alternate` values are ignored.

### CODE REVIEW

**Comment by kevinzetterstrom** [comment]
What about for values serialized before we had changed the names to hungarian? Is that an issue?

**Comment by anthonycr** [comment]
@kevinzetterstrom So in our app's version 2.4.0, we had some fields using hungarian notation, but most as not using it. As of our app's 2.5.0 version, all fields are using hungarian notation. Anything we serialized in version 2.4.0 without using the stag serializer will default to using the field name (camel case separated by underscores). Since our naming scheme was basically the reversal of that, we don't have to bother with fields that were not using hungarian notation (actually there are a few that I should fine).

Tl;dr; it's not an issue, except for a few stray fields who's keys don't match their variable names

**Comment by anthonycr** [comment]
I'm going to wait to merge this until some turnstile work is complete
